model/office/file_by_key.go

```diff
@@ -0,0 +1,44 @@
+package office
+
+import (
+	"bytes"
+
+	"github.com/cozy/cozy-stack/model/instance"
+	"github.com/cozy/cozy-stack/model/vfs"
+)
+
+func GetFileByKey(inst *instance.Instance, key string) (*vfs.FileDoc, error) {
```

Comment:
I find this function name a bit misleading as a conflict can be generated.

Reply:
Technically, the conflict has already happened, it was just not yet materialized on the Cozy. But, a file has been changed in OO and a new version has been uploaded, the stack will need to save the OO version in a new file. We just want to make it now, not wait for the OO callback.

Reply:
I understand the goal. I'm just saying that I would not expect an `io.cozy.file` to be created by calling a function named `GetFile…`.

Reply:
OK, I have renamed it to `EnsureFileForKey`